commit 56412894b3cee24805e48f380ffa9a5f32cff183
Author: Chao Yu <chao@kernel.org>
Date:   Mon Jun 12 22:30:44 2017 +0800

    f2fs: fix to document fault injection option and sysfs file
    
    Commit 73faec4d9935 ("f2fs: add mount option to select fault injection
    ratio") and Commit 087968974fcd ("f2fs: add fault injection to sysfs")
    forget to document mount option and sysfs file.
    
    This patch fixes to document them.
    
    Signed-off-by: Chao Yu <yuchao0@huawei.com>
    Signed-off-by: Jaegeuk Kim <jaegeuk@kernel.org>

diff --git a/Documentation/ABI/testing/sysfs-fs-f2fs b/Documentation/ABI/testing/sysfs-fs-f2fs
index 3cf9320f492c..b09108811ff1 100644
--- a/Documentation/ABI/testing/sysfs-fs-f2fs
+++ b/Documentation/ABI/testing/sysfs-fs-f2fs
@@ -112,3 +112,15 @@ Date:		January 2016
 Contact:	"Shuoran Liu" <liushuoran@huawei.com>
 Description:
 		 Shows total written kbytes issued to disk.
+
+What:		/sys/fs/f2fs/<disk>/inject_rate
+Date:		May 2016
+Contact:	"Sheng Yong" <shengyong1@huawei.com>
+Description:
+		 Controls the injection rate.
+
+What:		/sys/fs/f2fs/<disk>/inject_type
+Date:		May 2016
+Contact:	"Sheng Yong" <shengyong1@huawei.com>
+Description:
+		 Controls the injection type.